src/ports/postgres/modules/convex/test/mlp.sql_in

```diff
@@ -241,9 +247,8 @@ SELECT mlp_predict(
 
 select * from mlp_prediction;
 SELECT assert(
-    -- Accuracy greater than 90%
-    COUNT(*)/150.0 > 0.95,
-    'MLP: Accuracy is too low (< 95%). Wrong result.'
+    COUNT(*)/150.0 > 0.99,
```

Comment:
Tested on PG 9.4 and the latest commit of GPDB 5 (1 node, 3 segments).
IC passes on Postgres but fails on GPDB5. I tested the code manually and it works fine with 148/150 accuracy. I would suggest dropping the assertion threshold to 95% to make sure such minute changes do not fail the whole test.